mars/dataframe/core.py

```diff
@@ -1837,6 +1837,63 @@ def itertuples(self, index=True, name='Pandas', batch_size=1000, session=None):
         """
         return self._data.itertuples(batch_size=batch_size, session=session,
                                      index=index, name=name)
+    
+    def assign(self, **kwargs):
+        r"""
+        Assign new columns to a DataFrame.
+        Returns a new object with all original columns in addition to new ones.
+        Existing columns that are re-assigned will be overwritten.
+        Parameters
```

Comment:
Please pay attention to the blank line in docs, so do the other parts that I will not mention.